### ISSUE TITLE
What should we deliver for milestone 1?

### ISSUE THREAD

**Issue by michielbdejong** [federatedbookkeeping/task-tracking#7]
(continued from https://github.com/federatedbookkeeping/task-tracking/issues/4#issuecomment-1385248665):
I'm thinking maybe something like: pseudo code and elaborate and well-structured documentation (probably about 20 pages or so) for all the components involved in both timesheets and task tracking, with detailed lists of design choices and options.

**Comment by mcalligator** [comment]
Key outputs required for this milestone include:
- a refined definition of federation, taking into account privacy, transparency, integrity, data sharing mode (ad hoc, scheduled, live, etc.); and
- a protocol specification for federation.

**Comment by michielbdejong** [comment]
* different types of nodes
* different use cases
* different topologies

**Comment by mcalligator** [comment]
Yes, agreed!  Regarding use cases, I was thinking of including some high-level example scenarios as context for them, and providing some more detailed use cases, expanded further into user stories with acceptance criteria for specific functional deliverables implemented in code.

As for types of node,  I've captured those in draft in the [current definition of Federation](https://github.com/federatedbookkeeping/task-tracking/wiki/Milestone-1%3A-Theoretical-Groundwork/#system-federation) in the Wiki.

For topologies, we already described these in the Timesheets project, but it would be really good to build on that with the different real-world systems involved in this project, and depict these in a set up associated diagrams.

**Comment by michielbdejong** [comment]
I think we can do a lot of research already using just the use case that we have already partially satisfied with the timesheets network. A central question could be "how can we make the timeld gateway support two-way sync of multi-homed data?"
See https://github.com/federatedbookkeeping/task-tracking/issues/16#issuecomment-1426991674

**Comment by michielbdejong** [comment]
See https://matrix.to/#/!jlxDNfDSHXXyDmBKwk:gitter.im/$PirckqcPgxhOM1Rr29UWDw5sbZCge2v0-d0bIFBH28Q?via=gitter.im&via=matrix.org&via=coeus.ca

**Comment by michielbdejong** [comment]
I'm preparing an RfP for the Ponder Source side of milestone 1, here: #23

@mcalligator @gsvarovsky are you still planning to claim milestone 1b (975 euros) and your part (3413 euros) of milestone 1a? If so, do you want to coordinate or is it fine if we work in separate processes?

**Comment by mcalligator** [comment]
@michielbdejong it's been a while since we've had a chance to focus on this - could we maybe have a call to discuss, say, early next week?

**Comment by michielbdejong** [comment]
I'll email you

**Comment by michielbdejong** [comment]
RfP's for 1a and 1c sent.

I asked @mcalligator in the matrix channel what the status of 1b is.
